inyoka_theme_default/templates/portal/privmsg/index.html

```diff
@@ -0,0 +1,232 @@
+              <a{% if not entry.message.author.is_active %} class="user_inactive"{% endif %} href="{{ entry.message.author|url }}">
+              {{ entry.message.author|e }}
+              </a>
+            {% else %}
+              <a{% if not entry.message.recipients[0].is_active %} class="user_inactive"{% endif %} href="{{ entry.message.recipients[0]|url }}">
+              {{ entry.message.recipients[0]|e }}
+              </a>
+            {% endif %}
+            {% if entry.message.recipients|length > 1 %}, …{% endif %}
+          </td>
+          <td data-title="{% trans %}Date{% endtrans %}">{{ entry.message.pub_date|datetime }}</td>
+          <td data-title={% trans %}Actions{% endtrans %} class="actions">
+            {% if not entry.is_own_message %}
+              {% if entry.message.author.is_active %}
+                <a href="{{ entry|url('reply') }}" title="{% trans %}Reply{% endtrans %}">
+                  <span class="fa_icon-reply"></span>
+                </a>
+              {% endif %}
+              {% if entry.message.recipients|length > 1 %}
+                <a href="{{ entry|url('reply_to_all') }}" title="{% trans %}Reply to all{% endtrans %}">
+                  <span class="fa_icon-reply-all"></span>
+                </a>
+              {% endif %}
+            {% endif %}
+            <a href="{{ entry|url('forward') }}" title="{% trans %}Forward{% endtrans %}">
+             <span class="fa_icon-share"></span>
+            </a>
+            {% if not entry.in_archive %}
+              <a href="{{ entry|url }}?action=archive" title="{% trans %}Archive{% endtrans %}">
+                <span class="fa_icon-archive"></span>
+              </a>
+            {% endif %}
+          </td>
+          <td data-title="{% trans %}Delete{% endtrans %}" class="pn_delete">
+            <input type="checkbox" name="delete" value="{{ entry.id }}" />
+          </td>
+        </tr>
+        {% else %}
+        <tr>
+          <td colspan="6">
+            {% trans %}Currently there are no messages in this folder.{% endtrans %}
+          </td>
+        </tr>
+      {% endfor %}
+    </tbody>
+    <tfoot>
+      <tr>
+        <td colspan="6">
+          <div class="pnfooter">
+            {{ macros.render_pagination(pagination) }}
+            <p class="onepage">
+              {% if one_page %}
+                <a href="{{ href('portal', 'privmsg', folder.id) }}">
+                 {% trans %}Split messages into multiple pages{% endtrans %}
+                </a>
+              {% else %}
+                <a href="{{ href('portal', 'privmsg', folder.id, 'all') }}">
+                  {% trans %}Display all messages on one page{% endtrans %}
+                </a>
+              {% endif %}
+            </p>
+          </div>
+          <div class="linklist">
+            <a href="{{ href('portal', 'privmsg', 'new') }}" class="button">{% trans %}Compose message{% endtrans %}</a>
```

Comment:
i'm unsure atm whether this button is needed, as a button to compose a new message is also in the sidebar. So especially on desktops two buttons for the same action are side by side each other.